e2e/cypress/integration/integrations/integrations_spec.js

```diff
@@ -178,4 +182,25 @@ describe('Integrations page', () => {
         // * Validate that the correct empty message is shown
         cy.get('#emptySearchResultsMessage').should('be.visible').and('have.text', `No bot accounts match ${searchString}`);
     });
+
+    it('MM-T570 Integration Page titles are bolded', () => {
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Go to Main Menu -> Integrations
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+
+        cy.get('#integrations').should('be.visible').click();
```

Comment:
I'm good as is but would be better if
- scoped within a menu section (though we have this kind of approach in several places) and,
- if we can leverage [API queries from testing-library](https://testing-library.com/docs/dom-testing-library/api-queries) since it works in the perspective of test user rather than the DOM structure under the hood. Something like
```suggestion
        cy.get('.dropdown-menu').should('be.visible').findByText('Integrations').click();
```

Reply:
I'm not quite sure how to scope this within a menu section. Could you please clarify how that would look?

Reply:
Using the testing-library looks like a more maintainable approach. Thank you for introducing it to me!

Reply:
I missed your question. `cy.get('.dropdown-menu').should('be.visible').findByText('Integrations').click()` is one way to scope which is getting the dropdown menu and then finding "Integrations" in the menu. Another is you'll find the use of `.within(() => {})`

Reply:
Thanks for the explanation, I understand it now!